cmake/OpenFLUIDTestScript.cmake

```diff
@@ -38,6 +38,16 @@ FUNCTION(EXECUTE_COMMAND CMD)
     IF(NOT EXISTS ${ARGV1})
       MESSAGE(FATAL_ERROR "CHECK_FILE_EXIST: file ${ARGV1} not found")
     ENDIF()  
+
+  ELSEIF(${CMD} STREQUAL "CHECK_FILE_EXIST_IN_ARCHIVE")
+    SET(TMPDIR TMPDIR_UNZIPPED)
```

Comment:
Prefer generated name using UUID and/or timestamp (careful with possible same timestamp)

Reply:
Fixed by using RANDOM since UUID not fitting to use case and risks of identical timestamp